test/api/v3/unit/models/group.test.js

```diff
@@ -607,6 +613,90 @@ describe('Group Model', () => {
       });
     });
 
+    describe('#checkChatSpam', () => {
+      let testMessage, testUser, testTime;
+      let testUserID = '1';
+      beforeEach(async () => {
+        party._id = TAVERN_ID;
```

Comment:
I'd rather create a new variable than use `party` for this. It's confusing to look at.